backend/main/chapters/c06_lists.py

```diff
@@ -829,16 +826,367 @@ class sum_list(Step):
 - **`subscripting`**: Get a value at an index. `nums[0]` is 1, `nums[1]` is 2, `nums[2]` is 3.
 - **`+`**: Concatenates lists. `nums + [4, 5]` is `[1, 2, 3, 4, 5]`.
 
-Here's some new things. Try them out in the shell.
+Note that `nums.append(4)` modifies the existing list `nums`, while `nums + [4, 5]` does not.
+To preserve the value of `nums + [4, 5]` we must assign it to a *new variable*.
+Type and run the following code in the editor:
+
+__program_indented__
+        """
+
+        def program(self):
+            nums = [1, 2, 3]
+            new_nums = nums + [4, 5]
+            print(new_nums)
+            print(nums)
+            nums.append(4)
+            print(nums)
+
+    class pop_remove_index_subscript_assignment(ExerciseStep):
+        """
+As you see, `+` does not modify `nums`, but `append` does. (`new_nums` is not affected by the change in `nums`.)
+
+Here's some new things. Try them out in the shell. Again suppose we have a list `nums = [1, 2, 3]`.
```

Comment:
Once we merge https://github.com/alexmojaki/futurecoder/pull/73 it might be good to make actual steps of these with predicted outputs. In fact if we'd done that first we might not have done all this. But the combination is also good.

Reply:
Hmm, #73 is a complicated idea. I'm not sure what I think about it. It might be good if it's used sparingly.